about.html

```diff
@@ -0,0 +1,161 @@
+                    <img src="assets/jpg/kyle-smith-music-camp.jpg" alt="Image of GSMCT 2021 in Madrid">
+                    <div class="pe-text">
+                        <div class="year">2021</div>
+                        <div class="place">GSMCT 2021 in Madrid</div>
+                    </div>
+                   
+
+                </div>
+                <div class="past-event">
+                    <div class="pe-overlay"></div>
+                    <img src="assets/jpg/kyle-smith-music-camp.jpg" alt="Image of GSMCT 2020 in Helsinki">
+                    <div class="pe-text">
+                        <div class="year">2020</div>
+                         <div class="place">GSMCT 2020 in Helsinki</div>
+                    </div>
+                    
+                </div>
+            </div>
+        </section>
+        <section class="partners">
+            <h2 class="section-title">Partners</h2>
+            <div class="separator"></div>
+            <div class="partners-container">
+                <img src="images/partners/Sony-Music-Logo.svg" alt="Partner 1">
+                <img src="images/partners/Spotify_logo_with_text.svg" alt="Partner 2">
+                <img src="images/partners/Disney_wordmark.svg" alt="Partner 3">
+                <img src="images/partners/Deezer_logo.svg" alt="Partner 4">
+                <img src="images/partners/Virgin-logo.svg" alt="Partner 5">
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="footer-text">
+            <p>Made with <span class="heart"><i class="fas fa-heart"></i></span> by <a
+                    href="https://kingjosh007.github.io/my-portfolio/" target="_blank"
+                    rel="noreferrer noopener">KJC</a>, copyright © 2021</p>
+            <p>Original design idea by <a href="https://www.behance.net/adagio07" target="_blank"
+                    rel="noreferrer noopener">Cindy Shin</a> in <a
+                    href="https://www.behance.net/gallery/29845175/CC-Global-Summit-2015" target="_blank"
+                    rel="noreferrer noopener">Behance</a>.</p>
+        </div>
+        <div class="footer-icons">
+            <a href="https://github.com/Kingjosh007" target="_blank" rel="noreferrer noopener"><i
+                    class="fab fa-github"></i></a>
+            <a href="https://www.linkedin.com/in/king-josaphat-chewa-aa154011b" target="_blank"
+                rel="noreferrer noopener"><i class="fab fa-linkedin"></i></a>
+            <a href="https://angel.co/u/king-josaphat-nkouete-chewa" target="_blank" rel="noreferrer noopener"><i
+                    class="fab fa-angellist"></i></a>
+            <a href="https://twitter.com/KingJoChewa" target="_blank" rel="noreferrer noopener"><i
+                    class="fab fa-twitter"></i></a>
+            <a href="https://medium.com/@kingjochewa2007" target="_blank" rel="noreferrer noopener"><i
+                    class="fab fa-medium"></i></a>
+        </div>
```

Comment:
- [ ] The footer should contain a logo aligned to the left and text aligned to the right. Your design missed the logo. Always make sure you stick with the given design as it represents business/client requirements. Kindly compare your design with the original and make changes. 
- [ ] In addition, the footer has a dark background with white text for desktop view at least for the About page. 

![image](https://user-images.githubusercontent.com/64894927/132555535-700f772a-14e3-4285-a996-2f1ac21e2936.png)


![image](https://user-images.githubusercontent.com/64894927/132556774-dcffc0a1-2438-4066-b8f6-d12c439273f3.png)

Reply:
Sorry, I had to be absent from Thursday to Saturday last week. So I'm repeating the week after having used only a single attempt.
That change is done now.